modules/logging-loki-storage-azure.adoc

```diff
@@ -24,3 +24,26 @@ $ oc create secret generic logging-loki-azure \
   --from-literal=account_key="<azure_account_key>"
 ----
 <1> Supported environment values are `AzureGlobal`, `AzureChinaCloud`, `AzureGermanCloud`, or `AzureUSGovernment`.
+
+== Azure storage for STS enabled clusters
+
+If your cluster has STS enabled, the Cloud Credential Operator (CCO) supports short-term authentication using Azure AD Workload Identity.
+
+.Required data:
```

Comment:
As @xperimental suggested, these are the attributes on the secret created by the CCO and is internally used. I don't think we need to mention these here.

If we still want to, we should also specify the outline/sample of the secret created by the CCO and the description of those attributes (which are mentioned here)